packages/web/src/components/notification/components/desktop/Notification.tsx

```diff
@@ -67,9 +68,40 @@ const NotificationItem = (props: NotificationItemProps) => {
     user: props.user,
     users: props.users,
     entity: props.entity,
-    entities: props.entities
+    entities: props.entities,
+    value: 999,
+    reaction: 'fire'
   }
 
+  if (notification.type === NotificationType.RemixCosign) {
+    return <TipSentNotification notification={notification} />
+  }
+
+  // if (notification.type === NotificationType.Favorite) {
```

Comment:
probably need to clean up some test code/comments?

Reply:
oh yup good catch, this file wasn't supposed to be added!